zerberus/sfz.cpp

```diff
@@ -41,7 +42,7 @@ void SfzControl::init()
       {
       defaultPath.clear();
       octave_offset = 0;
-      defines.clear();
+      //defines.clear();
```

Comment:
If it's not needed, let's delete it and not comment it. It will raise questions in the future otherwise.